notebooks/official/model_evaluation/automl_text_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>notebook left hanging:</p><p> </p><ol><li>Showing that the new batch metrics where imported to the Model registry.</li><li>Showing how to retrieve either the training or batch metrics.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>